package/autoyast2.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           autoyast2
-Version:        3.1.101.1
+Version:        3.1.101.2
```

Comment:
3.1.Y is for master

3.1.Y.Z is for `released` products as Y is frozen

But we have already created 3.1.101.1 so it doesn't matter any more